packages/v4/patternfly-docs/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -7,7 +7,7 @@ A **tree view** is a structure that displays data in a hierarchical view. It can
 
 A default tree view provides a tree structure with items and arrows to expand or collapse child items. Every tree view can be extended with these [optional capabilities](#tree-view-capabilities).
```

Comment:
Need to re-link the "tree view capabilities" to say "tree view variations" and link to the variations heading instead